app/forms/steps/claim_details_form.rb

```diff
@@ -13,8 +13,8 @@ class ClaimDetailsForm < Steps::BaseFormObject
     validates :prosecution_evidence, presence: true
     validates :defence_statement, presence: true
     validates :number_of_witnesses, presence: true, numericality: { only_integer: true, greater_than: 0 }
-    validates :time_spent_hours, numericality: { only_integer: true, greater_than: 0 }
-    validates :time_spent_mins, numericality: { only_integer: true, greater_than: 0 }
+    # validates :time_spent_hours, numericality: { only_integer: true, greater_than: 0 }
+    # validates :time_spent_mins, numericality: { only_integer: true, greater_than: 0 }
```

Comment:
These should be optional validations - I was planning to mention I had spotted this issue on Friday.

i.e.
```ruby
validates :time_spent_mins, numericality: { only_integer: true, greater_than: 0 }, if: :preparation_time
```

Reply:
yep did try this witjh an unless but couldnt get it to work

Reply:
validates :time_spent_hours, numericality: { only_integer: true, greater_than: 0 }, if: :preparation_time
still doesnt work ?
in debugger when it is set to yes in persist
(ruby) preparation_time
#<YesNoAnswer:0x000000010a479e78 @value=:yes>
(ruby) preparation_time == true
false
(ruby) preparation_time.value == true
false

Reply:
validates :time_spent_mins, numericality: { only_integer: true, greater_than: 0 },
      if: ->(form) { form.preparation_time == YesNoAnswer::YES }